src/payment/data/service.js

```diff
@@ -36,6 +39,26 @@ export function configureApiService(newConfig, newApiClient) {
     }
     return response;
   });
+
+  const decodeURLParams = (search) => {
```

Comment:
Future thought:
I don't think it's necessary right now, but in future, we may want to forward other types of params as well.  For now, it seems UTM params aren't needed to be forwarded.